app/api/sessions.py

```diff
@@ -148,6 +150,11 @@ def before_update_object(self, session, data, view_kwargs):
         if session.is_locked and data.get('is_locked') == session.is_locked:
             raise ForbiddenException({'source': '/data/attributes/is-locked'}, "Locked sessions cannot be edited")
 
+        speakers_call = safe_query(self, SpeakersCall, 'event_id', session.event_id, 'event-id')
+        if speakers_call.ends_at.replace(tzinfo=None) < datetime.now().replace(tzinfo=None):
```

Comment:
No need to replace the `tzinfo`. Timezone does not play any role here.

Reply:
?
The speakers calls `ends at` is timezone aware
`  ends_at = db.Column(db.DateTime(timezone=True), nullable=False)`

Reply:
Suppose the event was created in the GMT timezone and you're in a CST timezone, would the speakers call time be independent of the timezone or would it be dependent when you're trying to access it?

Reply:
It should be dependent on my timezone when I am accessing it.

Reply:
It should be dependent. 

Like for GMT person - it ending at 5:00AM
Like for an CST person - it's ending at 3:00AM 

So for the CST person, editing is allowed < 3:00AM and for GMT people < 5:00 AM

Reply:
Yes, so why are you removing timezone from date?

Reply:
Yes, yes undoing it.